osc/babysitter.py

```diff
@@ -172,6 +172,10 @@ def run(prg, argv=None):
     except RPMError as e:
         print(e, file=sys.stderr)
     except SSLError as e:
+        if 'tlsv1' in str(e):
+            print('The API does not allow SSL Versions < TLSv1.2 and it\n' \
+                'ssems that you are running on a system with a python\n' \
```

Comment:
typo: seems

But I would not make it so vague. Just say that the system python version is not supporting TLSv1.2. 

In case the detection is wrong, it simply a bug ;)